pages/common/ghc.md

```diff
@@ -0,0 +1,23 @@
+# ghc
+
+> The Glassgow Haskell Compiler, compiles and links Haskell source files.
+
+- Find and compile all modules in the current directory:
+
+`ghc Main`
+
+- Compile a single file:
+
+`ghc {{file.hs}}`
+
+- Compile using optimization, for faster code:
+
+`ghc -O {{file.hs}}`
+
+- Stop after generating object files:
+
+`ghc -c {{file.hs}}`
+
+- Run in interactive mode (REPL):
```

Comment:
The reason is that the documentation calls it *"the interactive mode"* while the *"Haskell interpreter"* is actually a complete separate program. But I think i could try something else